entrypoint.sh

```diff
@@ -40,8 +40,11 @@ _git_is_dirty() {
     echo "INPUT_STATUS_OPTIONS: ${INPUT_STATUS_OPTIONS}";
     echo "::debug::Apply status options ${INPUT_STATUS_OPTIONS}";
 
+    echo "INPUT_FILE_PATTERN: ${INPUT_FILE_PATTERN}";
+    INPUT_FILE_PATTERN_ARRAY=( ${INPUT_FILE_PATTERN} )
```

Comment:
I think using this, expansion will happen:

```bash
> a="a b README.*";echo "a is: $a";b=( $a ); echo "b is: ${b[@]}"; 
a is: a b README.*
b is: a b README.md
```

You should use:

`readline -a INPUT_FILE_PATTERN_ARRAY <<< "${INPUT_FILE_PATTERN}"` 

demo:

```bash
> a="a b README.*";echo "a is: $a"; read -a b <<< "$a" ; echo "b is: ${b[@]}"; 
a is: a b README.*
b is: a b README.*
```

Reply:
It maybe because "<<<" does not compute path expansion:

from man bash:
```
   Here Strings
       A variant of here documents, the format is:

              [n]<<<word

       The word undergoes tilde expansion,  parameter  and  variable  expansion,  command  substitution,
       arithmetic  expansion,  and  quote  removal.   Pathname expansion and word splitting are not per‐
       formed.  The result is supplied as a single string, with a newline appended, to  the  command  on
       its standard input (or file descriptor n if n is specified).
```